src/helpers.js

```diff
@@ -62,3 +64,60 @@ export const openEditor = async () => {
     checkIsOpen()
   })
 }
+
+export const checkResourceUrlExists = async (url) => {
+  try {
+    const result = await fetch(url, {method: 'HEAD'})
+    return result.status === 200
+  } catch {
+    return false
+  }
+}
+
+export const checkExternalUrlExists = async (url) => {
+  try {
+    const command = `wget -q --method=HEAD ${url}`
+    const res = await window.codioIDE.remoteCommand.run(command)
+    return res.code === 0
+  } catch {
+    return false
+  }
+}
+
+export const promiseAllSeries = (arr) => {
```

Comment:
what is the difference between `Promise.all` and the function?

Reply:
Promise.all runs all request together. promiseAllSeries - one by one